src/openms/source/ANALYSIS/TOPDOWN/PeakGroup.cpp

```diff
@@ -172,22 +185,25 @@ namespace OpenMS
   }
 
   void PeakGroup::recruitAllPeaksInSpectrum(const MSSpectrum& spec, const double tol, const FLASHDeconvHelperStructs::PrecalculatedAveragine& avg, double mono_mass,
```

Comment:
please add comments in the code.
it is basically impossible to review without digging into the larger context.
At least higher level information (e.g., "Filter out blabalba") can help a lot.